### DISCUSSION TITLE
Home Assistant Core LXC - restore?

### DISCUSSION THREAD

**Discussion by tomasenskede** [community-scripts/ProxmoxVE#636] [General]
I am running Home Assistant in a Proxmox VM, and it works fine, except for occasional CPU spikes. I am exploring the possibility of moving to an LXC container, as I have had good experiences using LXC instead of a VM for several reasons.

Now, after running this excellent script, I have a functional Home Assistant LXC container :) but... the restore doesn't work. I transfer a full backup, and it shows up, but I can't restore it... why?
![image](https://github.com/user-attachments/assets/c5f7b8ec-a802-49d4-ba59-f68032debf15)

**Comment by Banjer** [comment]
Home Assistant **Core** supports making backups via the GUI, but restores need to be done manually.

https://community.home-assistant.io/t/guide-home-assistant-core-restoring-a-backup/626347

**Reply by tomasenskede** [reply]
I still haven’t been able to successfully migrate my VM-hosted Home Assistant to an LXC-hosted setup. I'm having trouble with the restore process — is there a way to do it manually? Are there any step-by-step guides available?

**Comment by Mellowlynx** [comment]
What backup are you restoring?
The small one is probably not correct.

Is there an error?

Also, to not get the CPU spikes in the VM, I found out that rebooting after an update works.
As I prefer the VM, as it has better hardware pass-through and the reboots works for me.
